src/main/java/Gumball/AddCommand.java

```diff
@@ -0,0 +1,51 @@
+package Gumball;
+
+import java.io.IOException;
+
+public class AddCommand extends Command {
+    private String input;
+
+    /**
+     *
+     * @param input The input that determines what task will be added.
+     */
+    public AddCommand(String input) {
+        this.input = input;
+    }
+
+    /**
+     *
+     * @param list The taskList where the task will be stored.
+     * @param ui Class which contains ui functions.
+     * @param fileManager The location where the information on the list is stored.
+     * @throws InputErrorException
```

Comment:
Perhaps can include more details on the circumstance under which this exception will be thrown e.g.
```suggestion
     * @throws InputErrorException when attempting to add an invalid <code>Task</code> object to the <code>TaskList</code>
```
Same for the `IOException` below and other comments